puzzles/puzzle1.html

```diff
@@ -130,79 +138,83 @@ <h1>Puzzle #1</h1>
       following exercise will have you add your own camper cell, html elements,
       and custom styling.
     </p>
+
     <p>
       <u><strong>Puzzle:</strong></u>
     </p>
+
     <ol>
-      <li>
-        In the cellPanel div, add your own div and give it a class of
-        'camperCell'
-      </li>
+      <li>In the cellPanel div, add your own div and give it a class of 'camperCell'</li>
       <li>Give your div a unique id</li>
       <li>Inside your div, add an h1 element that has your name</li>
       <li>Inside your div, add an image that has a picture/avatar of you</li>
-      <li>
-        Inside your div, add a p element that contains your favorite quote
-      </li>
-      <li>
-        Add any custom styling by targeting your custom id in the provided style
-        tag
-      </li>
+      <li>Inside your div, add a p element that contains your favorite quote</li>
+      <li>Add any custom styling by targeting your custom id in the provided style tag</li>
     </ol>
   </div>
 
   <div id="yourWork">
     <h1>Camper Cells</h1>
 
-    <div id="cellPanel">
+    <div id="cellPanel" class="row p-4">
```

Comment:
Great use of the bootstrap grid! This needed to be addressed and I'm glad you did it so cleanly!